site/docs/v1/tech/connectors/index.adoc

```diff
@@ -51,6 +51,13 @@ If you are using the generic connector, you are building the `user` object, so e
 
 In addition, ensure that the same `user.id` is returned each time a user is retrieved. If you have an LDAP attribute in the form of a UUID or a value that can be translated to a UUID, using that attribute or its translation is recommended. If you have to generate a UUID, ensure that you are able to return the same value each time the user authenticates. Doing so ensures that FusionAuth knows which user is being referenced, even if the username or email address changes.
 
+==== Suggested fields
+
+If you are authenticating a user, you likely want to populate these fields:
+
+* `user.registrations`: providing at least one entry in this array will associate a user with an application within FusionAuth, authorizing them to access the application.
+* `user.tenantId`: the tenant that this user should be associated with. Most useful when you have multiple tenants.
```

Comment:
Wouldn't this be dictated by the tenant that you're authenticating against?

Reply:
Good point. I tested and it seems that specifying a different tenant id than the tenant of the application being logged into is associated with doesn't work; the app's tenant wins.